switchlink/sai/switchlink_handle_lag.c

```diff
@@ -77,6 +77,11 @@ static int create_lag(const switchlink_db_interface_info_t* lag_intf,
       attr_list[ac].value.oid = *lag_h;
     }
     ac++;
+
+    attr_list[ac].id = SAI_LAG_ATTR_PORT_VLAN_ID;
```

Comment:
Please capture this as part of design document, we are using a different SAI attribute.